src/components/Viewer/BpmnViewer.tsx

```diff
@@ -198,31 +199,26 @@ const BpmnViewer: React.FC<Props> = props => {
                         }
 
                         if (props.showExpressions) {
-                            if (element.businessObject.extensionElements) {
-                                const extensionElements = element
+                            if (element.businessObject.extensionElements
+                                && element.businessObject.extensionElements.values) {
+                                const extensionElements: ModdleElement[] = element
                                     .businessObject.extensionElements.values;
-                                for (let j = 0; j < extensionElements.length; j++) {
-                                    if (extensionElements[j].$type === "camunda:executionListener") {
-                                        if (extensionElements[j].event === "end") {
-                                            overlays.add(element.id, "note", {
-                                                position: {
-                                                    bottom: 0,
-                                                    right: 0
-                                                },
-                                                html: `<div class='${classes.executionListener}'>${extensionElements[j].expression}</div>`
-                                            });
-                                        }
-                                        if (extensionElements[j].event === "start") {
-                                            overlays.add(element.id, "note", {
-                                                position: {
-                                                    bottom: 0,
-                                                    left: 0
-                                                },
-                                                html: `<div class='${classes.executionListener}'>${extensionElements[j].expression}</div>`
-                                            });
-                                        }
+                                extensionElements.forEach(extensionElement => {
+                                    const { $type, event, fields } = extensionElement;
+
+                                    if ($type === "camunda:ExecutionListener" && (event === "end" || event === "start") && fields) {
```

Comment:
Should we use .toLowerCase() for comparison? Or are you 100% sure it is spelled uppercase?

Reply:
.toLowerCase seems to be the safe way to go